addon/lib/document.js

```diff
@@ -127,6 +128,34 @@ export default Base.extend({
       console.debug(`${label}:`, any);
       return any;
     });
+    documentJexl.addTransform("min", (arr) => {
+      const nums = onlyNumbers(arr);
+      return nums.length ? Math.min(...nums) : null;
+    });
+    documentJexl.addTransform("max", (arr) => {
+      const nums = onlyNumbers(arr);
+      return nums.length ? Math.max(...nums) : null;
+    });
+    documentJexl.addTransform("round", (num, places) =>
+      isNaN(num)
+        ? null
+        : places
+        ? Math.round(num * Math.pow(10, places)) / Math.pow(10, places)
+        : Math.round(num)
+    );
+    documentJexl.addTransform("ceil", (num) =>
+      isNaN(num) ? null : Math.ceil(num)
+    );
+    documentJexl.addTransform("floor", (num) =>
+      isNaN(num) ? null : Math.floor(num)
+    );
+    documentJexl.addTransform("sum", (arr) =>
+      onlyNumbers(arr).reduce((num, base) => base + num, 0)
```

Comment:
Extract `.reduce((num, base) => base + num, 0)` into a `sum` function since you use that in the `avg` function as well